fe/src/main/java/org/apache/doris/analysis/DateLiteral.java

```diff
@@ -433,6 +433,14 @@ public static DateLiteral dateParser(String date, String pattern) throws Analysi
     //Return the date stored in the dateliteral as pattern format.
     //eg : "%Y-%m-%d" or "%Y-%m-%d %H:%i:%s"
     public String dateFormat(String pattern) throws AnalysisException {
+        //Replace java date style : {"yyyyMMdd", "yyyy-MM-dd", "yyyy-MM-dd HH:mm:ss"}
+        pattern = pattern.replaceAll("yyyy", "%Y").
+                replaceAll("MM", "%m").
+                replaceAll("dd", "%d").
+                replaceAll("HH", "%H").
+                replaceAll("mm", "%i").
+                replaceAll("ss", "%s");
```

Comment:
Is is a good idea to do like this?

1. If user put an format like "%dd" or "%d-dd-%m", you will change it format.
2. I think this modifing pattern work should be done by the client. For this case, it should be done by the from_unixtime function not here.